src/CoherenceBuild/Program.cs

```diff
@@ -1,233 +1,84 @@
-
-                    Retry(() =>
+                    new RepoInfo("UniverseCoherence", universeCoherenceDropDir.Value())
                     {
-
-                        var info = new PackageInfo
+                        PackagesToSkip =
                         {
-                            PackagePath = packageInfo.FullName,
-                            SymbolsPath = symbolsPath,
-                            IsCoreCLRPackage = isCoreCLR,
-                            IsCoherencePackage = isCoherencePackage,
-                            IsDnxPackage = isDnxPackage,
-                        };
-
-                        using (var reader = new PackageArchiveReader(packageInfo.OpenRead()))
+                            "MusicStore",
+                            "MusicStore.StandAlone"
+                        },
+                        FileSystemDependencies =
                         {
-                            info.Identity = reader.GetIdentity();
-                            info.PackageDependencyGroups = reader.GetPackageDependencies();
-                        }
-
-                        lock (dictionaryLock)
-                        {
-                            if (isCoreCLR)
-                            {
-                                processResult.CoreCLRPackages[info.Identity.Id] = info;
-                            }
-                            else
-                            {
-                                processResult.ProductPackages[info.Identity.Id] = info;
-                            }
-
-                            processResult.AllPackages[info.Identity.Id] = info;
-                        }
-                    });
-                });
-            }
-
-            return processResult;
-        }
-
-        private static string FindLatest(string repoDirectory, string buildBranch)
-        {
-            if (!Directory.Exists(repoDirectory))
-            {
-                return null;
-            }
-
-            return new DirectoryInfo(repoDirectory)
-                .EnumerateDirectories()
-                .Select(d =>
-                {
-                    int buildNumber;
-                    if (!int.TryParse(d.Name, out buildNumber))
+                            new FileDependency("commits") { Destination = "commits-universe" },
+                            new FolderDependecy(".build"),
+                        },
+                    },
+                    new RepoInfo("CoreCLR", coreCLRDropDir.Value())
                     {
-                        buildNumber = int.MinValue;
-                    }
+                        PackageDestinationDir = "ext",
+                        FileSystemDependencies =
+                        {
+                            new FolderDependecy("netcoresdk") { Optional = true }
+                        },
+                    },
+                    new RepoInfo("SignalR-Client-Cpp", "latest")
```

Comment:
The repo name used to be listed here - https://github.com/aspnet/Coherence/pull/96/files/3c319a0406dfb248d34139ae291d9648372f93ac#diff-941327d979d4d3fb4ad5068c28050da3L21. Unlike CoreCLR and UniverseCoherence, it's not really a part of a build chain. So we just pick up the latest drop here.